homeassistant/components/media_player/sonos.py

```diff
@@ -253,3 +258,14 @@ def turn_on(self):
     def play_media(self, media_type, media_id):
         """Send the play_media command to the media player."""
         self._player.play_uri(media_id)
+
+    def _is_reachable(self):
+        """Return True if player is reachable, False otherwise."""
+        sock = socket.socket(socket.AF_INET, socket.SOCK_STREAM)
+        try:
+            sock.connect((self._player.ip_address, 1443))
```

Comment:
I have been confused by the python 2 documentation:
> If no timeout is supplied, the global default timeout setting returned by getdefaulttimeout() is used.

However [python 3 documentation states](https://docs.python.org/3/library/socket.html#timeouts-and-the-connect-method):
> The connect() operation is also subject to the timeout setting, and in general it is recommended to call settimeout() before calling connect() or pass a timeout parameter to create_connection(). However, the system network stack may also return a connection timeout error of its own regardless of any Python socket timeout setting.

It looks like the timeout is already in place (according to my tests).

Nevertheless I can change the code to be something like: 

```python
create_connection(address=(self._player.ip_address, 1443), timeout=X)
```

What do you think?

What should I use as timeout value?